code/src/components/StartScreen.js

```diff
@@ -0,0 +1,72 @@
+import React, { useState } from "react";
+import { useDispatch } from "react-redux";
+import styled from "styled-components";
+import "nes.css/css/nes.min.css";
```

Comment:
Really cool that you used nes design! Seems like you didn't need to use that much code and still managed to make it look this good. Must try it next time!